recipe/meta.yaml

```diff
@@ -13,6 +14,9 @@ source:
 
 build:
   number: 100
```

Comment:
ping @mcg1969 - this makes me cringe as a method of determining which variant is installed.  Please try to find a better way.